### DISCUSSION TITLE
Web - Exception caught by scheduler library

### DISCUSSION THREAD

**Discussion by priyank-dt** [singerdmx/flutter-quill#170] [Web]
i'm using flutter quill latest version and Flutter 2.0.4 version in vscode.

we are building Flutter website so use this library but getting following error

════════ Exception caught by scheduler library ═════
Unexpected null value.
═════════════════════════════════════
![imgpsh_fullsize_anim](https://user-images.githubusercontent.com/82450392/114549006-207ff080-9c7e-11eb-8d90-f7ac9a53e3a9.png)

**Comment by florianh01** [comment]
Without additional information there is no way to track this issue.

**Reply by IntegroInfotech** [reply]
What additional Information u need to track this issue???

I used the example project getting same error.....

**Comment by GyuriMajercsik** [comment]
I'm getting something similar, but not sure if it is the same issue: 

```
======== Exception caught by foundation library ====================================================
The following TypeErrorImpl was thrown while dispatching notifications for FocusNode:
Unexpected null value.

When the exception was thrown, this was the stack: 
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/errors.dart 236:49      throw_
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/operations.dart 518:63  nullCheck
packages/flutter_quill/widgets/raw_editor.dart 952:25                             [_updateOrDisposeSelectionOverlayIfNeeded]
packages/flutter_quill/widgets/raw_editor.dart 963:5                              [_handleFocusChanged]
packages/flutter/src/foundation/change_notifier.dart 243:16                       notifyListeners
...
The FocusNode sending notification was: FocusNode#91dab([PRIMARY FOCUS])
  context: RawEditor-[LabeledGlobalKey<EditorState>#de492]
  PRIMARY FOCUS
====================================================================================================
```

**Comment by radoslaw-sz** [comment]
I have the same issue like @GyuriMajercsik .

**Reply by m-j-g** [reply]
> I have the same issue like @GyuriMajercsik .

Thats what I'm getting as well
